cinema_service/settings.py

```
@@ -86,11 +90,19 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ["DB_NAME"],
+        "USER": os.environ["DB_USER"],
+        "PASSWORD": os.environ["DB_PASSWORD"],
+        "HOST": os.environ["DB_HOST"],
+        "PORT": os.environ["DB_PORT"],
```

Database settings correctly use the PostgreSQL engine and fetch DB_* variables from environment. Ensure your docker-compose provides DB_NAME, DB_USER, DB_PASSWORD, DB_HOST and DB_PORT under the app environment (DB_HOST should usually be set to the service name `db`). If these are missing or DB_HOST is not set to the compose service name, the app will not connect to Postgres. Consider adding safe defaults for local development or a validation step to fail early with a clear message.